analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ParameterValidationInAsyncShouldBeWrapped.CSharp11.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace Tests.Diagnostics
+{
+    public interface IInvalidCases
```

Comment:
Discussed offline: These tests should stay.
There's an [issue](https://github.com/dotnet/runtime/issues/76673) during runtime regarding inheritance in interfaces - it's good to have tests to catch any possible regressions after its fixed (since roslyn might be affected).